### ISSUE TITLE
Trigger refresh when rebuilding Init containers

### ISSUE THREAD

**Issue by purkhusid** [GoogleCloudPlatform/freshpod#7]
Is it possible to trigger a refresh if a container has an init container and the init container is rebuilt?

**Comment by ahmetb** [comment]
I don't think we track initContainers today, but you can send a patch to support them easily!
